### PR TITLE
Fix quick start guide

### PR DESCRIPTION
**Pull request by vinnamkim**
### Summary
- Fix quick start guide according to the recent changes.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [ ] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: docs/source/docs/get-started/quick-start-guide/examples.rst [hunk 1]
```diff
@@ -29,13 +29,9 @@ Examples
...
     # Download COCO dataset http://cocodataset.org/#download
     # Put images to coco/images/ and annotations to coco/annotations/
-    datum create
-    datum import --format coco <path/to/coco>
-    datum export --filter '/image[images_I_dont_like]' --format cvat
-    # import dataset and images to CVAT, re-annotate
-    # export Datumaro project, extract to 'reannotation-upd'
-    datum project update reannotation-upd
-    datum export --format coco
+    datum project create
```
Comment (cih9088) [inline docs/source/docs/get-started/quick-start-guide/examples.rst:33]:
```suggestion
    mkdir my_project && cd my_project
    datum project create
```

File: docs/source/docs/get-started/quick-start-guide/examples.rst [hunk 2]
```diff
@@ -50,8 +46,8 @@ Examples
 
 .. code-block::
 
-    datum create
-    datum import --format coco <path/to/coco>
+    datum project create
```
Comment (cih9088) [inline docs/source/docs/get-started/quick-start-guide/examples.rst:51]:
```suggestion
    mkdir my_project && cd my_project
    datum project create
```

File: docs/source/docs/get-started/quick-start-guide/examples.rst [hunk 3]
```diff
@@ -63,8 +59,8 @@ Examples
 
 .. code-block::
 
-    datum create
-    datum import --format voc <path/to/voc/dataset>
+    datum project create
```
Comment (cih9088) [inline docs/source/docs/get-started/quick-start-guide/examples.rst:65]:
```suggestion
    mkdir my_project && cd my_project
    datum project create
```

**Review by wonjuleee — APPROVED** [review]
Great work!

**Review by cih9088 — COMMENTED** [review]
Since this is a quick start guide, users might just copy and paste the commands without knowing about messing up current directory by `datum project create`. How about adding creating project directory?

**Review by cih9088 — APPROVED** [review]

**Pull request merged**